test/global.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can we change this to msbuild? It's weird to have a repo that's partially csproj and project.json.